client/src/App.js

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-
-import LandingPage from './components/landingpage'
+import "./css/main.css";
+import LandingPage from "./components/LandingPage";
```

Comment:
This was the right move, but we need that change to be committed or everyone else will get an error when they spin this up.  You can change it with `git mv` like this: `git mv client/src/components/landingpage.js client/src/components/LandingPage.js`